komodo/symlink/sanity_check.py

```diff
@@ -82,10 +82,11 @@ def assert_root_nodes(link_dict):
     inferred_roots = _get_root_nodes(link_dict)
     if set(input_roots) != inferred_roots:
         raise AssertionError(
-            "The roots in the link-tree is not matching "
+            "The roots in the link-tree are not matching "
             + "the roots defined in link_roots in dict\n"
```

Comment:
The dict is called `root_links` I think? At least, that's what it's called in the symlink config JSON file in the `komodo-releases` repo.

So maybe, "the roots defined in the root_links dict" is better?

Reply:
Great catch!